material_components_ios.bzl

```diff
@@ -272,27 +272,42 @@ def mdc_unit_test_suite(
     minimum_os_version = IOS_MINIMUM_OS,
     visibility = ["//visibility:private"],
     size = "medium",
+    use_autobot_extension_runner = True,
```

Comment:
Is the `_extension` part necessary? We use this term to mean extensions to our components, so the use here may create some confusion.

Reply:
It is a good point. The reason for it is we need to add a suffix/prefix to the test name to avoid name conflict. And a iPhone X iOS 10 is used when not using autobot one, so it was name `extension` instead of `autobot` directly. How do you feel about `additional` name?

Reply:
Perhaps `_environment_runner`?

Reply:
Sounds good. Addressed in https://github.com/material-components/material-components-ios/pull/8259/commits/9f8ecf3eaf6eb2f6e057aa54f22dfccdd61e9b33.